platform/features/table/src/controllers/RTTelemetryTableController.js

```diff
@@ -69,53 +69,36 @@ define(
         RTTelemetryTableController.prototype = Object.create(TableController.prototype);
 
         /**
-         Override the subscribe function defined on the parent controller in
-         order to handle realtime telemetry instead of historical.
+         *
          */
-        RTTelemetryTableController.prototype.subscribe = function () {
-            var self = this;
-            self.$scope.rows = undefined;
-            (this.subscriptions || []).forEach(function (unsubscribe){
-                unsubscribe();
-            });
+        RTTelemetryTableController.prototype.addHistoricalData = function () {
```

Comment:
Agree, would be cleaner and was my original approach, but would require an 'abstract' TableController class that both RTTelemetryTableController and TelemetryTableController inherit from. Didn't want to add a third unnecessary class when we're just going to reconcile these anyway.